packages/docs/docs/concepts/cornerstone-tools/touch.md

```diff
@@ -0,0 +1,156 @@
+
+Other touch events that can occur are the `TOUCH_TAP` event. A `TOUCH_TAP` will not trigger a `TOUCH_START` event flow. If the user taps successively, only one `TOUCH_TAP` event will fire with the count of how many times the user tapped.
+
+| EVENT                  | Description                                                                                                                                                                                      |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `TOUCH_START`          | Triggers if the user places their touchpoint down for > 50ms.                                                                                                                                    |
+| `TOUCH_START_ACTIVATE` | Triggers only if no tools decided to stop propagation from the `TOUCH_START` event. It is useful to differentiate between touching an existing annotaton, vs needing to create a new annotaiton. |
+| `TOUCH_PRESS`          | Triggers if the user places their touchpoint down and does not move it for > 700ms                                                                                                               |
+| `TOUCH_DRAG`           | Triggers anytime the user moves their touchpoint, may occur before `TOUCH_PRESS` since the `TOUCH_PRESS` event will tolerate some motion.                                                        |
+| `TOUCH_SWIPE`          | Triggers alongside `TOUCH_DRAG` if the user moves more than `100px` within a single drag cycle.                                                                                                  |
+| `TOUCH_END`            | Triggers when the user lifts one or more of their touchpoints.                                                                                                                                   |
+| `TOUCH_TAP`            | Triggers when the user makes contact with screen for less than 50ms - 10ms (buffer from the `TOUCH_START` event.)                                                                                |
+
+## Multitouch
+
+Touch events natively support multitouch which is provided as a list of [`ITouchPoints[]`](api/tools/namespace/Types#ITouchPoints).
+In order for touch events to be compatiable with mouse events, these `ITouchPoints[]` need to be reduced into a single
+`ITouchPoint`. The current strategy for array reduction is taking the mean coordinate values. Other strategies can be
+implemented such as first point, median point, etc. This can be implemented in the
+[`touch` utilities codebase](https://github.com/cornerstonejs/cornerstone3D-beta/blob/gradienthealth/main/packages/tools/src/utilities/touch/index.ts)
```

Comment:
can we use main branch of the main repo instead of gradient health remote?

Reply:
yeah gradient health remote main should mirror cornerstonejs main. There were some weird rebase issues previously which I think are fixed now.

Reply:
Rename: 
`https://github.com/cornerstonejs/cornerstone3D-beta/blob/gradienthealth/main/packages/tools/src/utilities/touch/index.ts` => `https://github.com/cornerstonejs/cornerstone3D-beta/main/packages/tools/src/utilities/touch/index.ts`